docs/api/ipc-renderer.md

```diff
@@ -7,6 +7,15 @@ hide_title: false
 
 # ipcRenderer
 
+<!--
+```YAML history
+changes:
+  - pr-url: https://github.com/electron/electron/pull/40330
+    description: "`ipcRenderer` can no longer be sent over the `contextBridge`"
+    breaking-changes-header: behavior-changed-ipcrenderer-can-no-longer-be-sent-over-the-contextbridge
+```
+-->
```

Comment:
Nice. I think we might end up bikeshedding long-term over where this should go placement wise (after the quote, after the processes list, etc), but I like that we can document it here.

Could you add this example to the website PR just so we can see how it looks more easily?

Reply:
Added to website PR in [electron/website@`ca2f9de` (#594)](https://github.com/electron/website/pull/594/commits/ca2f9ded03b8d78bdffd14748b1826cec7a0b4cd)